src/istio/control/client_context_base.cc

```diff
@@ -93,7 +93,20 @@ ReportOptions GetReportOptions(const TransportConfig& config) {
   if (config.disable_report_batch()) {
     return ReportOptions(0, 1000);
   }
-  return ReportOptions();
+
+  // When batch reporting is enabled, if report_batch_max_entries or
+  // report_batch_max_time is set to 0 (default if not specified), set
+  // them to their default value defined in the ReportOptions constructor
+  uint32_t max_entries = config.report_batch_max_entries();
+  uint32_t max_time_ms = DurationToMsec(config.report_batch_max_time());
+
+  if (max_entries == 0)
+    max_entries = ::istio::mixerclient::DEFAULT_BATCH_REPORT_MAX_ENTRIES;
```

Comment:
Nit: always use bracket

Reply:
ok, let me make that change before merge. I'll take a look at envoy linter afterwards and see if I can copy its use in CI here.